sources/packages/backend/libs/test-utils/src/factories/student.ts

```diff
@@ -6,7 +6,10 @@ import { createFakeSINValidation } from "./sin-validation";
 import { getISODateOnlyString } from "@sims/utilities";
 // TODO: the parameter user must be moved to relations and all the references must be
 // updated.
-export function createFakeStudent(user?: User): Student {
+export function createFakeStudent(
+  user?: User,
+  options?: { initialValue: Partial<Student> },
```

Comment:
great to see the methods createFakeStudent and saveFakeStudent are enhancing like anything with the functionalities we create.